src/handlers/gestures/GestureDetector.tsx

```diff
@@ -644,6 +655,10 @@ export const GestureDetector = (props: GestureDetectorProps) => {
     applyEnableContextMenuProp(props.enableContextMenu, gestureConfig);
   }
 
+  if (props.touchAction !== undefined) {
+    applyTouchActionProp(props.touchAction, gestureConfig);
+  }
```

Comment:
We may want to figure out a better approach to do this, that's outside of the scope of this PR though 😄